src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/CpxVariantCanonicalRepresentation.java

```diff
@@ -339,9 +338,9 @@ static SimpleInterval getAffectedReferenceRegion(final List<SimpleInterval> even
             // it must be one of two cases:
             //   1) the base (and possibly following bases) immediately after (or before if reverse strand) the head alignment's ref span is deleted
             //   2) there are bases on the read immediately after the head alignment uncovered by selected alignments, i.e. unmapped insertion
-            final boolean firstSegmentNeighborsHeadAlignment = basicInfo.forwardStrandRep ? (firstSegment.getStart() - head.referenceSpan.getEnd() == 1)
-                                                                                          : (head.referenceSpan.getStart() - firstSegment.getEnd() == 1);
-            if ( ! firstSegmentNeighborsHeadAlignment )
+            final boolean expectedCase = basicInfo.forwardStrandRep ? (firstSegment.getStart() - head.referenceSpan.getEnd() == 1)
```

Comment:
The old name for this variable (`firstSegmentNeighborsHeadAlignment`) seemed more descriptive.

Reply:
reverted back